### PR TITLE
search field padding

### PR DESCRIPTION
**Pull request by vasilich6107**

### CODE REVIEW

**Comment by vasilich6107** [comment]
@salim-lachdhaf you've merged one but got two more ;-)

**Comment by vasilich6107** [comment]
Fixed confilcts

**Review by salim-lachdhaf — CHANGES_REQUESTED** [review]
`searchFieldPadding` should be an attribute of `searchFieldProps`

**Comment by vasilich6107** [comment]
fixed

**Review by salim-lachdhaf — CHANGES_REQUESTED** [review]

File: lib/src/text_field_props.dart [hunk 1]
```diff
@@ -43,6 +43,7 @@ class TextFieldProps {
     this.selectionWidthStyle = ui.BoxWidthStyle.tight,
     this.keyboardAppearance,
     this.scrollPadding = const EdgeInsets.all(20.0),
+    this.padding,
     this.dragStartBehavior = DragStartBehavior.start,
```
[the remark this answers is not in the document]
Comment (vasilich6107) [reply lib/src/text_field_props.dart:47]: done

**Pull request merged**